saltychat/NUI/SaltyWebSocket.html

```diff
@@ -10,7 +10,7 @@
 	</div>
 	
 	<script>
-        let pluginAddress = "127.0.0.1:8088";
+        let pluginAddress = "127.0.0.1:38088";
```

Comment:
This wouldn't work anyways since 127.0.0.1 is blocked by cfx and gets overwritten by the [script](https://github.com/saltminede/saltychat-fivem/blob/master/saltychat/SaltyClient/VoiceManager.cs#L104).
So it actually doesn't really matter what the inital value of this var is.
```suggestion
        let pluginAddress;
```